win32ss/gdi/ntgdi/freetype.c

```diff
@@ -160,6 +160,28 @@ static const CHARSETINFO g_FontTci[MAXTCIINDEX] =
     { SYMBOL_CHARSET, CP_SYMBOL, {{0,0,0,0},{FS_SYMBOL,0}} }
 };
 
+BYTE FASTCALL IntCharSetFromCodePage(UINT uCodePage)
+{
+    UINT i;
+
+    if (uCodePage == 1)  // CP_OEMCP
+        return OEM_CHARSET;
+
+    if (uCodePage == 2)  // CP_MACCP
```

Comment:
Isn't it possible to use the defines names directly?

Reply:
We cannot #include <winnls.h> at here, and if included <windows.h> then compilation will be slow...